FirebaseAuth/Sources/SystemService/FIRAuthStoredUserManager.m

```diff
@@ -81,6 +81,9 @@ - (FIRUser *)getStoredUserForAccessGroup:(NSString *)accessGroup
   query[(__bridge id)kSecAttrAccessGroup] = accessGroup;
   query[(__bridge id)kSecAttrService] = projectIdentifier;
   query[(__bridge id)kSecAttrAccount] = kSharedKeychainAccountValue;
+  if (@available(iOS 13.0, macOS 10.15, macCatalyst 13.0, tvOS 13.0, watchOS 6.0, *)) {
```

Comment:
From the [docs](https://developer.apple.com/documentation/security/ksecusedataprotectionkeychain): "The data protection key affects operations only in macOS. Other platforms automatically behave as if the key is set to true, and ignore the key in the query dictionary. You can safely use the key on all platforms."

Do you mind adding a comment with this information wherever `kSecUseDataProtectionKeychain` is used?

Reply:
@rosalyntan Will this be a problem if I did that (and rebase to resolve conflicts) beginning next week? Until that time, I only have access to the iPad, which is not a convenient tool for code editing and git operations.

Reply:
Thanks @kamilpowalowski It's fine to do next week. Our next release from master will be in January.

Reply:
Thanks for the additional cleanup @kamilpowalowski!

Reply:
Thank you, @kamilpowalowski! LGTM.